sdk/src/main/resources/sidechain-sdk-settings.conf

```diff
@@ -2,6 +2,13 @@ scorex {
   dataDir = /tmp/scorex/data/blockchain
   logDir = /tmp/scorex/data/log
 
+  logInfo {
```

Comment:
add the same block to `settings_basic.conf` and `sc_settings.conf`
even if it is a duplication with default values, but very useful to know for developers/users, because we refer to this files in our internal docs. So, they will see the these configs.

Reply:
Done